VPM.py

```diff
@@ -129,12 +132,10 @@ def compute_hop_length(win_length, overlap):
     Returns:
         hop_length (int): The computed hop_length.
     """
-    """
-    !! Write code here !!
-    """
+    return win_length*overlap
 
 # @Rachel/Shaun, this is the whole "Basic Preprocessing" part
-def stft(waveform, win_length=1024, overlap=.5, window='hann'):
+def stft(waveform, win_length=1024, hop_length=512, overlap=.5, window='hann', plot=True):
```

Comment:
```suggestion
def stft(waveform, win_length=1024, overlap=.5, window='hann', plot=True):
```
So my intention for not providing `hop_length` as an argument is so that we can tune the controls more easily using overlap % instead, which is more intuitive. Then, in the function, we use `hop_length = compute_hop_length(win_length, overlap)` to get the `hop_length`. If you think `hop_length` makes more sense than overlap, please add some checks, e.g. setting the default value of overlap to None, then `if overlap == None: ...`. But imo `hop_length` as an argument isn't very intuitive.